addons/product/models/product_pricelist.py

```diff
@@ -120,6 +120,7 @@ def _compute_price_rule(self, products, qty, uom=None, date=False, **kwargs):
             If not specified, prices returned are expressed in product uoms
         :param date: date to use for price computation and currency conversions
         :type date: date or datetime
+        :param bool rule_only: whether the price should be computed or not (default: False)
```

Comment:
I feel like this'd be clearer with something like `compute_price` instead of `rule_only`, but... :shrug:

Reply:
why not, ok then :D